norddrop/build.rs

```diff
@@ -126,6 +126,9 @@ fn main() -> Result<(), Box<dyn Error>> {
     if target_os == "windows" {
         create_winres(&version)?;
     }
+    if target_os == "android" {
+        println!("cargo:rustc-cdylib-link-arg=-Wl,-soname,libnorddrop.so");
```

Comment:
Can we construct `libnorddrop.so` from `CARGO_PKG_NAME` env ? In order to protect against renaming etc